test/support/model_case.ex

```diff
@@ -0,0 +1,20 @@
+defmodule JessicaBossWeb.ModelCase do
+  use ExUnit.CaseTemplate
+
+
+  using do
+    quote do
+      setup do
+        import Ecto.Query
+
+        alias JessicaBoss.Repo
+        alias JessicaBossWeb.UserModel
+
+        query = from u in UserModel.Schema
+        Repo.delete_all(query)
```

Comment:
Do we really need to do this? Ecto's sandbox should already give transactional connections for each test. Also, we're just handling the `User` model here. This won't scale if we start adding new models.

I think we just need to follow [these instructions](https://hexdocs.pm/ecto/Ecto.Adapters.SQL.Sandbox.html#module-example) and ensure that we're getting a new connection before each test:

```
setup do
  :ok = Ecto.Adapters.SQL.Sandbox.checkout(Repo)
  Ecto.Adapters.SQL.Sandbox.mode(Repo, {:shared, self()})
end
```

IMO, `:shared` is probably optional in our case. So:

```
defmodule JessicaBossWeb.ModelCase do
  use ExUnit.CaseTemplate

  setup do Ecto.Adapters.SQL.Sandbox.checkout(JessicaBoss.Repo) end
end
```